GTBitsOfGood/canine-assistants
# Madeline/199 edit placed dogs

## Fix Edit Placed Dogs Functions on Mobile + Bug Fixes

Issue Number(s): #199, #216

What does this PR change and why? adds ability to edit placed dogs on mobile and hides placed dog fields if location selection is not placed,. Also adds clearability to fields and makes form page back arrow clickable.

### Checklist


### Critical Changes

- None

### Related PRs

- None

### How To Test
Go to individual dog page on mobile and test if you can edit the dog. Also try selecting facilities for the location field and see that partner information should disappear and reappear upon selecting location = Placed.
